plugin/storage/es/factory.go

```diff
@@ -33,6 +33,11 @@ import (
 	"github.com/jaegertracing/jaeger/storage/spanstore"
 )
 
+const (
+	primaryNamespace = "es"
+	archiveNamespace = "es-archive"
```

Comment:
Can we change this to `es.archive` - reason being that the operator currently filters out storage specific options based on the first part ('.' separated) - and "es" is used for elasticsearch options.

Reply:
Isn't it an implementation detail of the operator?

I would like to be rather consistent with other storages configurations in Jaeger. Cassandra flags:
```
--cassandra-archive.tls
--cassandra.tls
```

Reply:
Yeah that is fine - was just thinking if only associated with elasticsearch options, then using a single part would help - but can be accommodated in the operator.

Reply:
Ok then I am resolving this, we will handle it in the operator.